lightning_transformers/core/loggers.py

```diff
@@ -0,0 +1,92 @@
+import time
+from typing import Dict, Optional, Union
+
+from pytorch_lightning.loggers import WandbLogger
+from sparseml.pytorch.utils.logger import LambdaLogger
+
+
+class WANDBLogger(WandbLogger):
```

Comment:
WANDB is not an abbreviation, so pls follow only the first letter convention

Reply:
Interesting, the convention actually seems to be to refer to weights & biases as wandb, but I'll change it to WABLogger to follow first letter convention